src/seismometer/seismogram.py

```diff
@@ -72,12 +71,21 @@ def __init__(self, config_path: Optional[str | Path] = None, output_path: Option
             config_path = Path(config_path)
 
         self.cohort_cols: list[str] = []
```

Comment:
Should this be redirected to config as well like you did for target_cols?

Reply:
The consistency would be nice, but I'm not sure this is the best PR.  
Would you want a separate issue written up for tracking?  And does the change fit with any of the control-viz changes you are doing?

I'm reluctant to change it here, when I know we'll be focused more deeply on handling cohorts and cohort attributes soon.

Reply:
Writing up as an separate issue for later resolution